receiver/mysqlreceiver/documentation.md

```diff
@@ -53,7 +55,10 @@ metrics:
 | buffer_pool_operations (operation) | The buffer pool operations types. | read_ahead_rnd, read_ahead, read_ahead_evicted, read_requests, reads, wait_free, write_requests |
 | buffer_pool_pages (kind) | The buffer pool pages types. | data, free, misc |
 | command (command) | The command types. | execute, close, fetch, prepare, reset, send_long_data |
+| digest (digest) | Digest. |  |
+| digest_text (digest_text) | Text before digestion. |  |
 | double_writes (kind) | The doublewrite types. | pages_written, writes |
+| event_states (kind) | Possible event states. | errors, warnings, rows_affected, rows_sent, rows_examined, created_tmp_disk_tables, created_tmp_tables, sort_merge_passes, sort_rows, no_index_used |
```

Comment:
Why `event_states` not `event_state`? can one metric have multiple states as an attribute? Maybe even just `state` given that it's always applied to metrics that have event in its name?

Reply:
I think `kind` is the more important value here, since that will be the actual metric attribute name. I think either `kind` or `type` is accurate and follows other common metric attribute names. 

That said, I think `event_states` should be changed to match and so should the description. Probably:
```suggestion
| event_type (type) | Possible event types. | errors, warnings, rows_affected, rows_sent, rows_examined, created_tmp_disk_tables, created_tmp_tables, sort_merge_passes, sort_rows, no_index_used |
```

Reply:
I will change it